client/src/pages/CreateWorkout/components/ExerciseCard/ExerciseCard.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { MdDeleteForever } from "react-icons/md";
+import { MdModeEdit } from "react-icons/md";
+
+const ExerciseCard = ({
+  exercise,
+  handleShowForm,
+  number,
+  handleEditExercise,
+}) => {
+  // will need a word to be empty, singular, or plural depending on the amount inputted by user
+  // ex. in regards to "sets" amount inputted by user. 0 = "", 1 = "set", >1 = "sets"
+  // pluralWordDesc must be a plural string
+  // using loose equality because input from user is in a string
+  const singularPluralOrEmpty = (amount, pluralWordDesc) => {
```

Comment:
This function can be outside this component since it doesn't depend on anything from the component besides what it takes in via the arguments. That way it won't get redefined everytime the component re-renders.